Air pollution prediction/training.py

```diff
@@ -0,0 +1,2 @@
+gpr = GaussianProcessRegressor(normalize_y=True).fit(X_train_df,y_train)
```

Comment:
You are importing/defining this GaussianProcessRegressor , X_train_df , y_train anywhere